compat/TESTING_GUIDE.md

```
@@ -0,0 +1,584 @@
+
+### 2.3 Phase 3: SSE 流式响应
+
+#### 验证方法 1: 单元测试 ✅
+
+```bash
+cd D:/cc/my-project/codex2api
+go test ./compat/sse -v
+```
+
+**测试结果**:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)